src/ui/widgets/DuelBarWidget/DuelRow.module.scss

```
@@ -0,0 +1,280 @@
+@media (prefers-reduced-motion: reduce) {
+  .slot,
+  .plate {
+    transition: none;
+  }
+}
+
+// A grid, not a flex row: every column has a width of its own and keeps it even
+// when what belongs in it is switched off, so nothing after a hidden distance
+// slides left. A plate whose columns resize with their content dances on every
+// tick, and a number that moves is a number you have to find again.
+//
```

Stylelint (17.14.0)

[error] Unexpected empty comment (scss/comment-no-empty)

(scss/comment-no-empty)